index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <main class="wrapper">
+      <header class="bar-nav">
+        <a href="index.html" class="logo">My Logo</a>
+        <nav><img src="image/Union.png" alt="Menu icon"></nav>
+      </header>
+      <div class="container">
```

Comment:
- [ ] Let's make sure this is a `section` tag better HTML semantic section tag should be the parent of div tag, not the other way around.

Reply:
I have updated it